packages/remediations/src/NewRemediationWizard.js

```diff
@@ -138,63 +91,109 @@ class RemediationWizard extends Component {
+            // eslint-disable-next-line camelcase
+            api.createRemediation({ name: formValues[SELECT_PLAYBOOK], add, auto_reboot: formValues[AUTO_REBOOT] }, state.basePath)
+            .then(({ id }) => resolver(id, formValues[SELECT_PLAYBOOK], true));
         }
-    }
+    };
 
-    render () {
-        return (
-            this.state.open ?
-                <FormRenderer
-                    schema={this.state.schema}
-                    container={this.container}
-                    subscription={{ values: true }}
-                    FormTemplate={this.getFormTemplate}
-                    initialValues={{
-                        [HAS_MULTIPLES]: !!this.state.resolutions?.find(r => r.resolutions.length > 1),
-                        [MANUAL_RESOLUTION]: true,
-                        [SELECTED_RESOLUTIONS]: {},
-                        [EXISTING_PLAYBOOK_SELECTED]: false
-                    }}
-                    componentMapper={{
-                        [componentTypes.WIZARD]: Wizard,
-                        ...this.state.mapperExtension
-                    }}
-                    onSubmit={(_, formOptions) => {
-                        this.onSubmit(formOptions.getState().values);
-                        this.setOpen(false);
-                    }}
-                    onCancel={this.closeWizard}
-                />
-                : <Fragment/>
-        );
-    }
-}
+    const mapperExtension = {
+        'select-playbook': {
+            component: state.errors.length > 0 ? FetchError : SelectPlaybook,
+            issues: data.issues,
+            systems: data.systems
+        },
+        'review-actions': {
+            component: ReviewActions,
+            issues: data.issues,
+            issuesMultiple: state.issuesMultiple
+        },
+        'issue-resolution': {
+            component: IssueResolution,
+            systems: data.systems,
+            resolutions: state.resolutions
+        },
+        review: {
+            component: Review,
+            data: data,
+            issuesById: state.issuesById,
+            resolutions: state.resolutions
+        }
+    };
+
+    return (
+        state.isOpen ?
+            <FormRenderer
+                schema={state.schema}
+                subscription={{ values: true }}
+                FormTemplate={(props) => <Pf4FormTemplate {...props} showFormControls={false} />}
+                initialValues={{
+                    [HAS_MULTIPLES]: !!state.resolutions?.find(r => r.resolutions.length > 1),
+                    [MANUAL_RESOLUTION]: true,
+                    [SELECTED_RESOLUTIONS]: {},
+                    [EXISTING_PLAYBOOK_SELECTED]: false
+                }}
+                componentMapper={{
+                    [componentTypes.WIZARD]: Wizard,
+                    ...mapperExtension
+                }}
+                onSubmit={(_, formOptions) => {
+                    onSubmit(formOptions.getState().values);
```

Comment:
You get values as the first argument (`_`)

Reply:
But in the first argument, there are not all the values, just these mentioned in the schema :/

Reply:
Yep, there are only values that you visited in the wizard.

Reply:
I don't see values that I set manually using formOptions.change()
Is it ok to use the second param or can I add it somehow to the first one?

Reply:
As we talked in DMs, the only concern is too handle values that users do not visit. If you have managed it elsewhere, then it's no issue to use the function.